corehq/ex-submodules/casexml/apps/case/mock/tests/test_case_block.py

```diff
@@ -39,6 +39,22 @@ def test_does_not_let_you_specify_a_keyword_twice(self):
             ).as_xml()
         self.assertEqual(six.text_type(context.exception), "Key 'case_name' specified twice")
 
+    def test_let_you_specify_system_props_for_create_via_updates(self):
+        actual = ElementTree.tostring(CaseBlock.deprecated_init(
```

Comment:
I realize other tests around here are using it, but why not have this new test not use deprecated behavior?
```suggestion
        actual = ElementTree.tostring(CaseBlock(
```